tls-client/main.cpp

```diff
@@ -443,6 +443,8 @@ int main() {
         return 1;
     }
 
+    // The platform context is passed just in case any crypto calls need it.
+    // Please refer to https://github.com/ARMmbed/mbedtls/issues/1200 for more information.
```

Comment:
Please move this to the `HelloHTTPS` ctor, and have same comment like in the `hashing` and `benchmark`